### PR TITLE
feat(multiframe-pointpainting): add multi-sweep pointpainting

### PR DESCRIPTION
**Pull request by tzhong518**
Signed-off-by: tzhong518 <sworgun@gmail.com>

## Description

- add multi-sweep pointpainting.
- the old PR includes unneccessary commits and became messy, so we decided to create a new PR for the changes.
- rosbag with yolo's result: https://drive.google.com/drive/folders/1cqXByizuWT22nYx9j_5fB42lyGJk9pzg?usp=sharing

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/image_projection_based_fusion/src/utils/geometry.cpp [hunk 1]
```diff
@@ -122,7 +122,7 @@ void boundingBoxToVertices(
 
   const auto position = Eigen::Vector3d(pose.position.x, pose.position.y, pose.position.z);
   const auto orientation = Eigen::Quaterniond(
-    pose.orientation.w, pose.orientation.x, pose.orientation.y, pose.orientation.z);
+    pose.orientation.x, pose.orientation.y, pose.orientation.z, pose.orientation.w);
```
Comment (yukkysaito) [inline perception/image_projection_based_fusion/src/utils/geometry.cpp:125]: Is it correct?
https://eigen.tuxfamily.org/dox/classEigen_1_1Quaternion.html

Comment (tzhong518) [reply perception/image_projection_based_fusion/src/utils/geometry.cpp]: fixed by retrieve new commit

**Comment by yukke42** [comment]
@tzhong518 Cloud you add pointpainting option to `tier4_perception_launch` ?
ref: https://github.com/tier4/autoware_launch/pull/426

**Review by miursh — APPROVED** [review]
LGTM
I verified that it works correctly.
image

**Review by yukke42 — APPROVED** [review]
LGTM

**Pull request merged**